openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -691,35 +691,35 @@ If the Access Token is valid for requesting issuance of multiple Credentials, it
 
 ## Binding the Issued Credential to the Identifier of the End-User Possessing that Credential {#credential-binding}
 
-Issued Credential SHOULD be cryptographically bound to the identifier of the End-User who possesses the Credential. Cryptographic binding allows the Verifier to verify during the presentation of a Credential that the End-User presenting a Credential is the same End-User to whom that Credential was issued. For non-cryptographic type of binding and Credentials issued without any binding, see Implementation Considerations in (#claim-based-binding) and (#no-binding).
+An issued Credential SHOULD be cryptographically bound to the identifier of the End-User who possesses the Credential. Cryptographic binding allows the Verifier to verify during the presentation of a Credential that the End-User presenting a Credential is the same End-User to whom that Credential was issued. For non-cryptographic types of binding and Credentials issued without any binding, see the Implementation Considerations in (#claim-based-binding) and (#no-binding).
 
-Note: Claims in the Credential are usually about the End-User who possesses it, but can be about another entity.
+Note: Claims in the Credential are about the subject of the Credential, which is often End-User who possesses it.
```

Comment:
```suggestion
Note: Claims in the Credential are about the subject of the Credential, which is often the End-User who possesses it.
```

Reply:
@tplooker your suggested change (adding "the") was already in place.  I think someone else found it first.